sql-plugin/src/main/java/com/nvidia/spark/rapids/ColumnarCopyHelper.java

```diff
@@ -18,6 +18,11 @@
 
 import ai.rapids.cudf.HostColumnVector.ColumnBuilder;
 
+import ai.rapids.cudf.HostColumnVectorCore;
+import org.apache.hadoop.hive.ql.exec.vector.DecimalColumnVector;
+import org.apache.spark.sql.execution.datasources.orc.OrcAtomicColumnVector;
+import org.apache.spark.sql.execution.datasources.orc.OrcColumnVector;
```

Comment:
nit: I don't think that the ORC columns are used and should probably be removed.

Reply:
Yes, I fixed it.